### ISSUE TITLE
Some parts of App Center do not scale text correctly with "Large Text" mode enabled

### ISSUE THREAD

**Issue by shytzedaka** [ubuntu/app-center#1403]
When using the accessibility mode "Large Text" as presented here:
![image](https://github.com/ubuntu/app-center/assets/20986867/6ffb8d89-d132-40c1-8ae5-0c113aa2f656)
or using the terminal command to get the same effect: 
`gsettings set org.gnome.desktop.interface text-scaling-factor 1.25`

Text does not appear to be extending or scaling in some areas of the App Center as can be seen here:
![image](https://github.com/ubuntu/app-center/assets/20986867/4b346364-4b2f-4eb2-9e69-17b9c82f40f1)

it appears the issue only occurs at the sidebar of the app center and not the rest of the program.
**Note: I am Using a 1920x1080 Display with regular 100% scale and the App Center is fully maximized**
![image](https://github.com/ubuntu/app-center/assets/20986867/63fa795b-48d4-4a40-a957-23674cad1d15)

**Comment by d-loose** [comment]
Thanks for the screenshots!
We definitely need to improve the spacing in some places to accommodate larger fonts. A few more screenshots (with ratings backend running):

|Normal Font|Large Font|
|-|-|
|![Screenshot from 2023-09-25 13-56-22](https://github.com/ubuntu/app-center/assets/113362648/8eabb653-d58d-4d9e-8e56-ade0a2e24485)|![Screenshot from 2023-09-25 13-56-14](https://github.com/ubuntu/app-center/assets/113362648/1b821a3a-03d6-48b1-a030-19a873d2c097)|
|![Screenshot from 2023-09-25 13-57-48](https://github.com/ubuntu/app-center/assets/113362648/9d3367df-c0fd-44a8-ac12-aa012f453e56)|![Screenshot from 2023-09-25 13-58-02](https://github.com/ubuntu/app-center/assets/113362648/c626f3e7-0a5d-4bf0-b74a-cf223311ba09)

@anaserejio: something to look at when you got time :)

**Comment by anorld-droid** [comment]
@d-loose I have improved the spacings.
![Screenshot from 2023-10-06 11-01-21](https://github.com/ubuntu/app-center/assets/61829388/e6b0cdc0-96f7-4b8c-8a2f-12598f5c3b23)
![Screenshot from 2023-10-06 11-01-21](https://github.com/ubuntu/app-center/assets/61829388/90165c22-0d41-43b3-aa7d-c58cbd7606a9)

**Comment by anorld-droid** [comment]
What is the expected outcome for this, should we increase it's width?
https://user-images.githubusercontent.com/113362648/270333836-c626f3e7-0a5d-4bf0-b74a-cf223311ba09.png

**Comment by d-loose** [comment]
> @d-loose I have improved the spacings.

Thanks - looks great! I'm wondering if there's enough space for the ratings though. Can you check what it looks with the ratings enabled?
```
RATINGS_SERVICE_URL=ratings.ubuntu.com flutter run
```

**Comment by d-loose** [comment]
> What is the expected outcome for this, should we increase it's width? https://user-images.githubusercontent.com/113362648/270333836-c626f3e7-0a5d-4bf0-b74a-cf223311ba09.png

I don't know yet, we'll need to patiently wait for @anasereijo, who works on the design for the app center.

**Comment by anorld-droid** [comment]
> > @d-loose I have improved the spacings.
> 
> Thanks - looks great! I'm wondering if there's enough space for the ratings though. Can you check what it looks with the ratings enabled?
> 
> ```
> RATINGS_SERVICE_URL=ratings.ubuntu.com flutter run
> ```

Yeah, not so good. We may have to increase the height of the card to accommodate the summary;it has a maxlines of 2.

![image](https://github.com/ubuntu/app-center/assets/61829388/59673f70-d08d-4e8c-99f3-0ca9f6b3942d)

**Comment by anasereijo** [comment]
Hi @shytzedaka, thank you for filling the issue. We are planning on doing an accessibility assessment on the app, but in the meantime, feel free to file any issues or propose improvements. :)

@anorld-droid thanks for doing this 🙌  one question: are you changing the height of the cards when Large text is enabled or for both situations (enabled and disabled)?

**Comment by anorld-droid** [comment]
> Hi @shytzedaka, thank you for filling the issue. We are planning on doing an accessibility assessment on the app, but in the meantime, feel free to file any issues or propose improvements. :)
> 
> @anorld-droid thanks for doing this 🙌 one question: are you changing the height of the cards when Large text is enabled or for both situations (enabled and disabled)?

Both, but I will add when Large text is enabled; I had not thought of that.

**Comment by shytzedaka** [comment]
hey! it seems that this bug so far persists in released Ubuntu 23.10.1
I'm not sure if this is still being fixed or not but i thought it would be worth pointing that out since we're out of the beta phase.

![image](https://github.com/ubuntu/app-center/assets/20986867/51fc9504-ed71-4e20-86ef-bdacb59b853a)

**Comment by shytzedaka** [comment]
Thank you for your hard work :)

**Comment by frankhuurman** [comment]
![image](https://github.com/user-attachments/assets/139483a6-b9bf-443d-a2ea-89a650f44f69)
![image](https://github.com/user-attachments/assets/32036bfb-dc76-410a-98c3-8bbc7c148aa4)

Just want to chime in that even without Large Text mode or any sort of scaling enabled it can't spell "Manage" completely on a 1080p screen even though it looks like there's plenty of space.
Just upgraded to 24.04.1 from 22.04.4, not sure if it's something on my end or happens for others as well.

**Comment by frankhuurman** [comment]
@d-loose ah perfect, thanks for the quick reply Dennis :)
